mirage/config.js

```diff
@@ -325,6 +325,33 @@ export default function () {
     return this.serialize(builds, 'build');
   });
 
+  this.post('/repo/:repo_id/requests', function (schema, request) {
```

Comment:
This doesn't seem like a very typical use case, so probably fine to make it a one-off response with raw JSON. That said, it might be good to add a mirage model/serializer if we're going to work with these endpoints more in the future.